resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/kubernetes/submit/submitsteps/initcontainer/InitContainerConfigurationStepsOrchestratorSuite.scala

```diff
@@ -77,7 +151,7 @@ class InitContainerConfigurationStepsOrchestratorSuite extends SparkFunSuite {
     val orchestrator = new InitContainerConfigurationStepsOrchestrator(
       NAMESPACE,
       APP_RESOURCE_PREFIX,
-      SPARK_JARS,
+      SPARK_JARS.take(1),
```

Comment:
this is a bit brittle in case SPARK_JARS changes in the future -- we should create a new `SPARK_JARS_REMOTE` that has only `hdfs://` path